module/VuFind/src/VuFind/Cover/Router.php

```diff
@@ -76,12 +88,38 @@ public function getUrl(RecordDriver $driver, $size = 'small')
             return false;
```

Comment:
I understand now why you were failing out of the AJAX method when getThumbnail returned an empty result -- it was the parallel to this logic. We should probably make the logic align in both places, but at this point I'm not sure whether it's better to remove this logic or add it back to the AJAX handler. (And apologies for any confusion I'm causing -- it's been a while since I refreshed my memory on how this all works).

Reply:
I've reviewed the logic again, and it looks fine to me now; resolving this thread.